_extras/guide.md

```diff
@@ -27,13 +27,14 @@ is included as [`code/gen_inflammation.py`](../code/gen_inflammation.py).
 ## Overall
 
 This lesson is written as an introduction to Python,
-but its real purpose is to introduce the single most important idea in programming:
-how to solve problems by building functions,
-each of which can fit in a programmer's working memory.
-In order to teach that,
-we must teach people a little about
-the mechanics of manipulating data with lists and file I/O
-so that their functions can do things they actually care about.
+but its wider purpose is to introduce an important idea in programming:
+how to solve problems by building functions, units of code that allow
+reusability and efficient logical organization of software.
```

Comment:
I don't really think the sentence needs to change further, but if you are wanting to break it up and change it, I prefer "However" over "Still" and a `,` instead of "and" between "efficient" and "logical". However, I am fine with Maxim's suggestion as-if if that is what you all want. @ndilekli what please commit as you see fit!

Reply:
I'm OK if we don't break it into two but I think we need to add a comma or an `and` between "efficient" and "logical" and maybe change comma dashes before "the units". The reason I proposed to break this sentence up is that it seems to have too many "stops" / "breaks":
1. `, but`
2. `programming:`
3. `functions,`
4. `efficient [and/,] logical`

... and breaking up the sentence into two had minimal effect on the changes in this PR. Other way is to remove "to introduce an important idea in programming:" so that it reads

> but its wider purpose is to show how to solve problems by building functions...

but this is open up for a debate and, again, we can always leave it as-is here and address it in a future PR.

Reply:
Resolving in favor of https://github.com/swcarpentry/python-novice-inflammation/pull/837#discussion_r450963457